### PR TITLE
chore: Enable jumpcloud adapter for dev and staging

### PR DESCRIPTION
**Pull request by jamilbk**
Will need to do the same for prod after deploy.

### CODE REVIEW

**Review by bmanifold — APPROVED** [review]
LGTM

File: terraform/environments/staging/portal.tf [hunk 1]
```diff
@@ -263,7 +263,7 @@ locals {
     # Auth
     {
       name  = "AUTH_PROVIDER_ADAPTERS"
-      value = "email,openid_connect,google_workspace,token,microsoft_entra,okta"
```
Comment (AndrewDryga) [inline terraform/environments/staging/portal.tf:266]: Let's add it to `production/portal.tf` right away?

Comment (jamilbk) [reply terraform/environments/staging/portal.tf:266]: Will that cause an issue if the adapter is not deployed?

Comment (AndrewDryga) [reply terraform/environments/staging/portal.tf:266]: Nope, it's a whitelist: https://github.com/firezone/firezone/blob/andrew/bugfixes-11/elixir/apps/domain/lib/domain/auth/adapters.ex#L33

**Pull request merged**